balancer-js/examples/data/token-yields.ts

```diff
@@ -10,11 +10,12 @@ const { data } = sdk;
 
 const tokens = [
   yieldTokens[1].waDAI,
-  '0x93ef1ea305d11a9b2a3ebb9bb4fcc34695292e7d', // qETH
+  '0x93ef1Ea305d11a9b2a3ebb9bb4fcc34695292e7d', // qETH
```

Comment:
was that an issue somewhere? we should always use lowercase addresses

Reply:
No, not at all, I just changed to test if it was working if the user inputs an address with an uppercase letter. Forgot to change back to lowercase😅